packages/@aws-cdk/aws-ecs/test/ec2/ec2-service.test.ts

```diff
@@ -54,9 +54,36 @@ describe('ec2 service', () => {
 
       expect(service.node.defaultChild).toBeDefined();
 
-
     });
 
+    test('token param', () => {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const vpc = new ec2.Vpc(stack, 'MyVpc', {});
+      const cluster = new ecs.Cluster(stack, 'EcsCluster', { vpc });
+      addDefaultCapacityProvider(cluster, stack, vpc);
+      const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'Ec2TaskDef');
+
+      taskDefinition.addContainer('web', {
+        image: ecs.ContainerImage.fromRegistry('amazon/amazon-ecs-sample'),
+        memoryLimitMiB: 512,
+      });
+
+      const service = new ecs.Ec2Service(stack, 'Ec2Service', {
+        cluster,
+        taskDefinition,
+      });
+
+      expect(service.serviceArn).toContain(`service/${cluster.clusterName}/${service.serviceName}`);
+
+      const importService = ecs.Ec2Service.fromEc2ServiceAttributes(stack, 'importService', {
+        cluster: cluster,
+        serviceArn: service.serviceArn,
+      });
+      expect(importService.serviceArn).toContain(`service/${cluster.clusterName}/${service.serviceName}`);
+      expect(importService.serviceName).toEqual(service.serviceName);
```

Comment:
this should be it's own test once the fromXXX() issues are fixed in https://github.com/aws/aws-cdk/pull/18140